src/libprojectM/Common.hpp

```diff
@@ -61,24 +61,6 @@ extern FILE *fmemopen(void *buf, size_t len, const char *pMode);
 #define STRING_LINE_SIZE 1024
 
 
-#ifdef __unix__
-#include <cstdlib>
-#define projectM_isnan std::isnan
-#endif
-
-#ifdef EMSCRIPTEN
-#include <cstdlib>
-#define projectM_isnan isnan
-#endif
-
-#ifdef WIN32
-#define projectM_isnan(x) ((x) != (x))
-#endif
-
-#ifdef __APPLE__
-#define projectM_isnan(x) ((x) != (x))
-#endif
-
 #ifdef __unix__
 #define projectM_fmax fmax
```

Comment:
In a later revision, we could also get rid of those and replace calls with `std::max<float>` and `std::min<float>`, respectively.